src/services/validation.js

```diff
@@ -0,0 +1,22 @@
+import firebaseAPI from "@/services/API/checkUser.js";
+import { sendPlio } from "./sendPlio";
+
+export async function validateUser(userID, validateCount, isSingleEntryOnly, redirectID, doesUserExist){
```

Comment:
The name should be something like `validateSRNUser` no?
Because slowly we'll add more functions in this file for different kinds of validation right?

Reply:
Yeah I realized that and changed it in a later commit.